### ISSUE TITLE
Orders Not displaying

### ISSUE THREAD

**Issue by yousufkhalid223** [bagisto/bagisto#5137]
Hi,

I recently setup the bagisto on my local system. I am checking the system and found an issue in it that, i place an order via guest and after that when i go to admin panel for checking the order the order page only showing blank nothing there but when i export the orders the order is visible on export sheet but not on browser.

**Comment by devansh-webkul** [comment]
Hi there,

This issue has been fixed in master. Please take a pull from the master.

**Comment by derMart** [comment]
Hey, I do have the same issue. I am running version 1.3.3 which was release on Sep. 27, much later than your reply from Sep. 06, so I would assume the fix is already included in that relase.
Is that true, so should the problem already be fixed in v1.3.3 ?

**Comment by derMart** [comment]
Addition: I do see the following error in the javascript console:

[Vue warn]: Unknown custom element: <datagrid-plus> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

(found in <Root>)

**Comment by devansh-webkul** [comment]
I think something wrong with your instance. You can verify this in the demo also because orders table is visible.

**Comment by derMart** [comment]
thnx for the answer, yes in deed I found the problem. JS cache had an old version of some .js file, clearing cache solved the problem. Followup: We did upgrade following https://devdocs.bagisto.com/1.x/introduction/upgrade-to-latest-bagisto.html#bagisto-changes from 1.3.1 to 1.3.3
Is it possible that .js files did not get updated this way and that for an upgrade one needs to rebuild js files using npm install && npm run development / production in the respective package directories? These steps are not provided in the upgrade guide but I could image they are important, otherwise leading to such cache problems?
